components/standings/standings_storage.lua

```diff
@@ -0,0 +1,36 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Standings/Storage
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local StandingsStorage = {}
+
+function StandingsStorage.run(index, data)
+	mw.ext.LiquipediaDB.lpdb_standing(
+		'standing' .. index,
+		{
+			title = data.title,
+			tournament = data.tournament,
+			participant = data.participant,
+			participantdisplay = data.participantdisplay,
+			participantflag = data.participantflag,
+			icon = data.icon,
+			placement = data.placement or data.rank,
+			definitestatus = data.definitestatus or data.bg,
+			currentstatus = data.currentstatus or data.pbg,
+			change = data.change,
+			scoreboard = mw.ext.LiquipediaDB.lpdb_create_json({
+				match = data.match, -- [won, draw, lost]
+				overtime = data.match, -- [won, draw, lost]
```

Comment:
```suggestion
				overtime = data.overtime, -- [won, draw, lost]
				game = data.game, -- [won, draw, lost]
```
* typo for overtime
* add game scores, which def is needed on several wikis

Reply:
What should game scores look like?

Reply:
i assume there might be wikis that allow draws in games and still count them towards the match scores, so probably the same as matches/overtime
(for sc2 we would only need win/loss)

Reply:
Done